manifests/0000_90_dns-operator_02_servicemonitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Where is dns operator runlevel in the boot order compared to monitoring? In ingress-operator, we manage servicemonitor (and the other resources introduced in this PR) from the operator itself because of bootstrap ordering issues around this specific resource.

Reply:
According to the latest documentation, everything runs at the same time: https://github.com/openshift/cluster-version-operator/commit/18a9ab85abd09fddcdc3777ab6fe8d74f060228b#diff-c69c529fb860f12ea2dc7781e702b5fcR15

Reply:
Will the absence of the ServiceMonitor CRD affect dns-operator's rollout in any way?

Reply:
It should not.  Several operators define `ServiceMonitor` resources in their manfiests:
```
% print -l *operator/manifests/*servicemonitor.yaml(:h)
cluster-authentication-operator/manifests
cluster-dns-operator/manifests
cluster-image-registry-operator/manifests
cluster-kube-apiserver-operator/manifests
cluster-kube-controller-manager-operator/manifests
cluster-kube-scheduler-operator/manifests
cluster-openshift-apiserver-operator/manifests
cluster-openshift-controller-manager-operator/manifests
cluster-openshift-controller-manager-operator/manifests
%
```